content/blog/2016/2016-02-24-jenkins-security-update.md

```diff
@@ -0,0 +1,12 @@
+---
+:layout: post
+:title: Jenkins security updates
+:tags:
+- general
+- core
+- jenkinsci
+:author: daniel-beck
+---
+We released Jenkins updates today that include important security fixes: 1.650 and 1.642.2. For detailed information about the security content of these updates, see the [security advisory](https://wiki.jenkins-ci.org/display/SECURITY/Jenkins+Security+Advisory+2016-02-24).
```

Comment:
Makes sense to specify versions of bundled components with security fixes, so maintainers of custom cores will be able to pick them up